introduction.Rmd

```diff
@@ -524,7 +535,7 @@ step, Stan computes the gradient of the objective function, which is
 used in the "leapfrog algoritm" to determine the direction in which
 the algorithm moves (see Section 12.4 of _Bayesian Data Analysis_ and
 the "MCMC Sampling" chapter of the _Stan Reference Manual_).
-An iteration might have tens or hundreds of steps.
+A iteration might have tens or hundreds of steps.
```

Comment:
"An" is correct - the "n" is required before a vowel, furthermore, the rule is phonetic, not orthographic, so if the initial consonant is silent, e.g. "hour", you also write "an hour".
https://www.grammar.com/a-vs-an-when-to-use/